k8s/calibrate/prod-airqo-calibrate-app.yaml

```diff
@@ -36,7 +36,7 @@ spec:
       containers:
         - name: prod-calibrate-app
           image: eu.gcr.io/airqo-250220/airqo-calibrate-app:latest
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
[only pull images when they are not present locally](https://github.com/airqo-platform/AirQo-frontend/pull/1058/commits/ca5f1784b33cfe016f1fc163df49149babcc7c84)